packages/kit/src/exports/public.d.ts

```diff
@@ -846,7 +846,7 @@ export interface Navigation {
 	/**
 	 * The type of navigation:
 	 * - `form`: The user submitted a `<form>`
-	 * - `leave`: The user is leaving the app by closing the tab or using the back/forward buttons to go to a different document
+	 * - `leave`: The user is leaving the app by closing the tab or using the back/forward buttons to go to a different document, or as a result of programmatic external navigation such as `location.href = ...`
```

Comment:
this is kind of long. it's also probably not exhaustive - if the user types in another URL via the URL bar then that would be included as well I think?
```suggestion
	 * - `leave`: The app is being left either because the tab is being closed or a navigation to a different document is occurring
```